devel/bun/bun/Portfile

```
@@ -0,0 +1,23 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem  1.0
+PortGroup npm 1.0
+
+name                bun
+version             1.1.10
+maintainers         johnlindop.com:git
+revision            0
+
+description         JavaScript runtime built from scratch to serve the modern JavaScript ecosystem
+
+long_description    Bun is an all-in-one JavaScript runtime & toolkit designed for \
+                    speed, complete with a bundler, test runner, and Node.js-compatible \
+                    package manager
+
+categories          devel
+homepage            https://bun.sh
+license             MIT
+
+checksums           rmd160  b7c118e527f587e5832b7b096780d60d6120bfc6 \
+                    sha256  17e4deb24dc9d328c5a5b37fc1e58187e1d8587134378e9abf7e5f94379bf2f9 \
+                    size    5455
```

Unix text files must end with a newline.

```suggestion
                    size    5455

```